test/performance/benchmarks_test.go

```diff
@@ -108,17 +111,15 @@ func runTest(t *testing.T, img string, baseQPS float64, loadFactors []float64) {
 		t.Fatal("No result found for the load test")
 	}
 
-	if resp.ErrorsPercentage(0) > 0 {
-		t.Fatal("Found non 200 response")
-	}
-
-	// Add latency metrics
 	var tc []junit.TestCase
+	// Add latency metrics
 	for _, p := range resp.Result[0].DurationHistogram.Percentiles {
```

Comment:
yeah it seems we can't still separate success latencies from failure latencies.

Reply:
Not in this change, but I can check if there is the way to separate them. Ideally there should be an option in `fortio`  to rule out the failing requests when collecting the latency data.

Reply:
May be we should switch to @mattmoor 's new darling load test framework: https://github.com/tsenart/vegeta

Reply:
even that looks like it provides metrics over all requests. `mean is the arithmetic mean / average of the latencies of all requests in an attack.`